### ISSUE TITLE
New directions

### ISSUE THREAD

**Issue by avdi** [avdi/ppwm#80]
Thanks @hanneskaeufler for nudging me about this project 😉 While bringing the content up to date, I feel like it might also be time to reconsider the tech and hosting. Last I recall there isn't actually anything dynamic about this site - should it be a Jekyll/Hugo site hosted on GitHub pages?

**Comment by bf4** [comment]
anything that people maintaining are willing to do is ok by me :)

**Comment by hanneskaeufler** [comment]
👍 Thanks for the invite! I thought about a migration to a static site as well, but haven't quite looked into the "matcher" functionality yet. It is included as a rails engine as far as I can seen (which is also the reason for me not having the PR to upgrade to rails 3 ready).

I guess I will push forward on the following, in order of "most benefit to the user".

* content
* responsive layout
* more up to date technical stack (might either be upgrading or moving to another one)

I assume the site is deployed to heroku on push to master @avdi, right? Is that currently giving you pain/cost?

**Comment by avdi** [comment]
Yeah that's the current state of it, and that might actually explain some of the Heroku "over your limit" emails I've been getting 

—
Avdi Grimm

--- original message ---
On January 10, 2020, 11:41 AM CST notifications@github.com wrote:


👍 Thanks for the invite! I thought about a migration to a static site as well, but haven't quite looked into the "matcher" functionality yet. It is included as a rails engine as far as I can seen (which is also the reason for me not having the PR to upgrade to rails 3 ready).

I guess I will push forward on the following, in order of "most benefit to the user".

content

responsive layout

more up to date technical stack (might either be upgrading or moving to another one)


I assume the site is deployed to heroku on push to master @avdi, right? Is that currently giving you pain/cost?

—

You are receiving this because you were mentioned.

Reply to this email directly, view it on GitHub, or unsubscribe.
--- end of original message ---
